core/src/main/java/hudson/slaves/ChannelPinger.java

```diff
@@ -136,6 +150,7 @@ public void onClosed(Channel channel, IOException cause) {
         });
 
         t.start();
-        LOGGER.fine("Ping thread started for " + channel + " with a " + interval + " minute interval");
+        LOGGER.fine("Ping thread started for " + channel + " with a " +
+            intervalSeconds + " second interval and a " + timeoutSeconds + " second timeout.");
```

Comment:
What do you mean by slow logging? Building up the string like that (including in case of FINE not actually being logged)?